week-3/F-array-forEach/exercise.js

```diff
@@ -9,6 +9,25 @@
 
 var arr = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
 
+
+function multiplesFizzBuzz(element) {
+  for (var i = 1; i < arr.length; i++) {
+    if (arr[i] % 3 == 0 && arr[i] % 5 == 0) {
+      arr[i] = "FizzsBuzz";
+    }
+    if (arr[i] % 3 == 0) {
```

Comment:
Unfortunately this won't work well since you don't have "else" instead of "else if" in L18 and L21.
For example, for element 15, L16, L19, and L22 will be executed, which means it will be transformed to "Buzz" instead of the correct "FizzBuzz"